admin/app/components/solidus_admin/ui/table/component.html.erb

```diff
@@ -103,7 +103,7 @@
               content_tag(:span, "0", "data-#{stimulus_id}-target": "selectedRowsCount"),
               " #{t('.rows_selected')}.",
             ])), colspan: @columns.count - 1) %>
-          </div>
+          </tr>
```

Comment:
Ha!

Reply:
I was impressed to get just one issue, but I think we ran it in the past and already fixed whatever we caught 🙌

Reply:
Yeah!